example/lambdas/python-processing/pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[project]
+name = "cumulus-python-processing"
+version = "21.0.1"
+description = "Python reference activity that utilizes the Cumulus Process SDK."
+requires-python = "~=3.12.0"
+
+dependencies = [
+    "boto3~=1.40.29",
+    "cumulus-process==1.6.0",
+    "cumulus-message-adapter-python==2.4.0",
+    "six>=1.17.0",
+]
+
+[dependency-groups]
+dev = [
+    "mypy~=1.19.0",
+    "pytest~=9.0.2",
+    "pytest-mock~=3.15.1",
+    "ruff~=0.14.0",
+]
+
+[tool.ruff]
+extend = "../../../pyproject.toml"
```

Comment:
I don't think it's necessary to specify the `extend` unless you're trying to override something. By default ruff will pick up config files only if the have a `tool.ruff` section, so if we don't specify here, it should use the root one automatically. (As far as I understand it) https://docs.astral.sh/ruff/configuration/#config-file-discovery


```suggestion
```

Reply:
Thanks! Removed from each of the toml files I created.